monitoring/statuscake/variables.tf

```diff
@@ -25,3 +25,23 @@ variable "confirmation" {
   description = "Retry the check when an error is detected to avoid false positives and micro downtimes"
   default     = 2
 }
+
+# Heartbeat check variables
+variable "enable_heartbeat" {
```

Comment:
There is no need for this variable since heartbeat_names is a list. Heartbeat will automatically be disabled when the list is empty